changes/cmd/changetool/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/aggagen/changes"
+	"os"
+)
+
+func usage() {
+	fmt.Printf("USAGE: todo")
+}
+
+func main() {
+	metadata, err := changes.LoadMetadata()
+	if err != nil {
+		fmt.Printf("Failed to load .changes directory: %v\n", err)
+		os.Exit(1)
+	}
+
+	addFlags := flag.NewFlagSet("add", flag.ExitOnError)
+	addModule := addFlags.String("module", "", "creates a change for the specified module")
+
+	lsFlags := flag.NewFlagSet("ls", flag.ExitOnError)
+	lsModule := lsFlags.String("module", "", "filters changes by module")
+
+	if len(os.Args) < 2 {
+		usage()
+		os.Exit(1)
+	}
+
+	switch os.Args[1] {
+	case "add":
+		addFlags.Parse(os.Args[2:])
+		addCmd(metadata, *addModule)
+	case "ls", "list":
+		lsFlags.Parse(os.Args[2:])
+		lsCmd(metadata, *lsModule)
+	case "modify", "edit":
+		if len(os.Args) < 3 {
+			usage()
+			os.Exit(1)
+		}
+
+		modifyCmd(metadata, os.Args[2])
+	case "rm", "delete":
+		if len(os.Args) < 3 {
+			usage()
+			os.Exit(1)
+		}
+
+		rmCmd(metadata, os.Args[2])
+	case "release":
+		fmt.Println(metadata.CreateRelease("2020-06-19", []changes.VersionBump{}))
```

Comment:
Should release date be a flag argument that defaults to the value for todays date?

Reply:
Yeah I think so. The release command is currently more of a placeholder that I was using to test some of the Metadata operations.